### PR TITLE
✨ Launch the 'use responsive ads for responsive sizing in auto ads' experiment to a 5% holdback

### PR DESCRIPTION
**Pull request by riklund**
See https://github.com/ampproject/amphtml/issues/24168.

### CODE REVIEW

**Comment by riklund** [comment]
@lannka @keithwrightbos @charliereams @tlong2

File: extensions/amp-auto-ads/0.1/placement.js [hunk 1]
```diff
@@ -216,7 +216,7 @@ export class Placement {
           return this.state_;
         }
         const useResponsiveAdElement =
-          isResponsiveEnabled && this.responsiveSizingBranch_ == '368226531';
+          isResponsiveEnabled && this.responsiveSizingBranch_ != '368226533';
```
Comment (lannka) [inline extensions/amp-auto-ads/0.1/placement.js]: can we import the value? easy to forgot updating this line later

Comment (riklund) [reply extensions/amp-auto-ads/0.1/placement.js]: Not easily - importing it seemed to create a circular dependency.

Comment (lannka) [reply extensions/amp-auto-ads/0.1/placement.js]: then the value declaration should be moved here?

**Pull request merged**